tests/basic/autopilot_test.go

```diff
@@ -1848,6 +1848,204 @@ var _ = Describe(fmt.Sprintf("{%sRebalanceWithApproval}", testSuiteName), func()
 	})
 })
 
+// This testsuite for cases including: pvc resize in large scale, pvc resize for sharedv4 volume
+// executing pool reblance and expansion at the same time
+var _ = Describe(fmt.Sprintf("{%sFunctionalTests}", testSuiteName), func() {
+	var testrailID = 12345
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/12345
+	var runID int
+	JustBeforeEach(func() {
+		StartTorpedoTest(fmt.Sprintf("{%sFunctionalTests}", testSuiteName), "Perform several autopilot functional tests", nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+
+	It("has to run rebalance and resize pools, validate rebalance, validate pools and teardown apps", func() {
+		var contexts []*scheduler.Context
+		testName := strings.ToLower(fmt.Sprintf("%srebalance", testSuiteName))
+		poolLabel := map[string]string{"autopilot": "resizedisk"}
+		storageNodes := node.GetStorageDriverNodes()
```

Comment:
This will return all the nodes with storageDriver installed. Do we want that? If you need only storagenodes, there is a method: `node.GetStorageNodes()` will return only nodes which are part of storage pool

Reply:
I think it's needed because the job is to expand pool that uses cloud drive. 
Tried to change to GetStorageNodes in my test branch (https://github.com/portworx/torpedo/blob/ezhang/test/tests/basic/autopilot_test.go#L1866) and rerun the pipeline, it failed due to errors like 

> device path is not present for volume: pvc-c3ec5c90-1205-4c1a-bdac-1630604d11e0

https://jenkins.pwx.dev.purestorage.com/job/Autopilot/job/tp-autopilot-functional-tests/16/consoleFull